lib_xua/src/core/endpoint0/xua_endpoint0.c

```diff
@@ -887,6 +907,7 @@ void XUA_Endpoint0_loop(XUD_Result_t result, USB_SetupPacket_t sp, chanend c_ep0
                             outuint(c_aud_ctl, AUDIO_STOP_FOR_DFU);
                             // Handshake
                             chkct(c_aud_ctl, XS1_CT_END);
+                            notify_audio_stop_for_DFU = 1;
```

Comment:
As discussed this can be fixed by moving https://github.com/xmos/lib_xua/blob/5634172a19d63ab65f95f8c2dd20dec2afdf1810/lib_xua/src/core/audiohub/xua_audiohub.xc#L923 to the DummyDeliver loop

Reply:
As discussed the list of != commands would be better as an equality check with flash accessing commands (dnload, upload etc)

Reply:
Fixed this.